accounts/models.py

```diff
@@ -85,6 +85,15 @@ class Profile(SocialModel):
     def __unicode__(self):
         return self.user.username
 
+    def has_many_emails(self, return_qs=False):
+        """Check if the user account associated with this profile
```

Comment:
Now it always returns queryset. I think in your initial implementation it returned boolean, I added the qs option at some point and I guess later modified some of your original code so the boolean option was in fact not used anymore. I know the name of the method is not perfect if it returns a qs, but I think it makes the code readable so I've kept it.
https://github.com/MTG/freesound/pull/763/commits/2f4f6718dc9246d17e9ee4576fa7caefa0883879